<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultipleSet</class>
 <widget class="QWidget" name="MultipleSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="7">
    <widget class="QPushButton" name="pb_saveInfo">
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;设备型号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comB_deviceType">
       <item>
        <property name="text">
         <string>仪器1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>仪器2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>仪器3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>仪器4</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2" colspan="6">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>收发信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="te_showMsg">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pb_start">
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>IP地址</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLineEdit" name="le_ip1"/>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="styleSheet">
          <string notr="true">font: 24pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_ip2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="styleSheet">
          <string notr="true">font: 24pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_ip3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="styleSheet">
          <string notr="true">font: 24pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_ip4"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选中表格某行数据进行开始</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="6">
    <widget class="QTableWidget" name="table_Info">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>仪器名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>IP地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>1211数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>1240数量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>总次数</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>失败次数</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="pb_stop">
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QPushButton" name="pb_deleteSet">
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
